.github/workflows/test.yml

```diff
@@ -162,15 +162,20 @@ jobs:
   # 4. E2E - Insurance
   e2e-tests-insurance:
     name: E2E Insurance 👷
-    needs: setup
+    needs: [setup, unit-tests, api-tests]
     environment:
       name: ${{ needs.setup.outputs.environment }}
     runs-on: ubuntu-latest
 
     strategy:
       fail-fast: true
+      # Single dimension matrix - 4 jobs
       matrix:
-        machines: [1, 2, 3, 4, 5, 6, 7, 8]
+        machines: [1, 2, 3, 4]
```

Comment:
Does halving the machines increase the time it takes to run the tests? I think we currently have just under 300 insurance E2E tests.

Reply:
Testing various combinations.